docs/make.md

```diff
@@ -30,6 +30,10 @@ Building
 Image building
   build-podman          Build container image using Podman
 
+Dashboard
+  generate-dashboard    Generate dashboard
+  validate-dashboard    Compare dashboard configmaps with git
+
```

Comment:
Interesting, this passed CI/CD but now it bugs me. Probably there is a bug in the checker.